modules/swagger-codegen/src/test/java/io/swagger/codegen/options/JaxRSServerOptionsProvider.java

```diff
@@ -25,7 +25,9 @@ public String getLanguage() {
 
         ImmutableMap.Builder<String, String> builder = new ImmutableMap.Builder<String, String>();
         builder.putAll(options)
-                .put(CodegenConstants.IMPL_FOLDER, IMPL_FOLDER_VALUE);
+                .put(CodegenConstants.IMPL_FOLDER, IMPL_FOLDER_VALUE)
+                //.put(JavaJaxRSJersey1ServerCodegen.DATE_LIBRARY, "joda") //java.lang.IllegalArgumentException: Multiple entries with same key: dateLibrary=joda and dateLibrary=joda
```

Comment:
@hiveship given that JavaJaxRSJersey1ServerCodegen no longer exists, I'll remove this commented line.